doc/getting_started/how_to_renate_config.rst

```diff
@@ -114,6 +114,19 @@ These are optional as well but, if omitted, Renate will use :code:`train_transfo
     :caption: Example
     :lines: 75-
 
```

Comment:
you will need to update the line since you've added more lines to the beginning. please also check if this file was used somewhere else